console/src/assets/i18n/en.json

```diff
@@ -620,58 +624,58 @@
             }
         },
         "NAME": "Name",
-        "TYPE":"Application type",
+        "TYPE":"Application Type",
         "GRANT":"Grant Types",
         "OIDC": {
             "PROSWITCH":"I'm a pro. Skip this wizard.",
             "NAMEANDTYPESECTION":"Name and Type",
-            "TITLEFIRST":"Insert a name first!",
+            "TITLEFIRST":"Insert a name first.",
             "TYPETITLE":"What type of application do you want to create?",
-            "REDIRECTTITLE":"Specify the urls where the login will redirect to.",
-            "POSTREDIRECTTITLE":"This is the redirect after logout.",
-            "REDIRECTDESCRIPTIONWEB":"Redirects must begin with https://. http:// is only valid with enabled dev mode.",
-            "REDIRECTDESCRIPTIONNATIVE":"Redirects must begin with your own protocol or http://localhost!",
-            "REDIRECTNOTVALID":"This redirect is not valid!",
-            "TYPEREQUIRED":"The type is required!",
+            "REDIRECTTITLE":"Specify the URIs where the login will redirect to.",
+            "POSTREDIRECTTITLE":"This is the redirect URI after logout.",
+            "REDIRECTDESCRIPTIONWEB":"Redirect URIs must begin with https://. http:// is only valid with enabled development mode.",
+            "REDIRECTDESCRIPTIONNATIVE":"Redirect URIs must begin with your own protocol or http://localhost.",
+            "REDIRECTNOTVALID":"This redirect URI is not valid.",
+            "TYPEREQUIRED":"The type is required.",
             "TITLE": "OIDC Configuration",
             "CLIENTID": "Client ID",
             "CLIENTSECRET": "Client Secret",
-            "CLIENTSECRET_DESCRIPTION": "Save your Client Secret as it will disappear once the dialog is closed!",
+            "CLIENTSECRET_DESCRIPTION": "Keep your client cecret at a safe place as it will disappear once the dialog is closed.",
```

Comment:
```suggestion
            "CLIENTSECRET_DESCRIPTION": "Keep your client secret at a safe place as it will disappear once the dialog is closed.",
```

Reply:
Please change `cecret` to `secret`.